pkg/cmd/gc/gc_gke.go

```diff
@@ -507,12 +507,16 @@ func (o *GCGKEOptions) getServiceAccounts() ([]serviceAccount, error) {
 		return nil, err
 	}
 
+	var modifiedSAs []serviceAccount
 	for _, sa := range serviceAccounts {
-		if sa.DisplayName == "" {
+		// If there isn't a display name or the display name contains spaces, which is the case for Terraform-created
```

Comment:
I think the better way to do this is anyways to use the email. That is something in a fixed known format. The display name is probably one of the worst things to use in this case.